content/en/dev-tut-dvm-withdraw.md

```diff
@@ -0,0 +1,31 @@
+---
+id: dev-tut-dvm-withdraw
+title: Withdraw from DVM address
+sidebar_label: Withdraw from DVM address
+---
+
+> The DVM address format is identical to the Ethereum address format
+
+With the DVM smart contract solution, users can easily use ethereum contracts in the Darwinia Network, as well as the
+commonly used tools MetaMask and Remix, etc. DVM account withdrawal is the act of withdrawing tokens from a DVM account
+to a Darwinia account.
+
+Materials that need to be prepared in advance:
+
+- MetaMask(with dvm account imported)
+- Apps, which integrated withdraw tool
+- Withdrawal address: A Darwinia account
+
+Withdraw Process:
+
+1. Connect Apps with MetaMask
+
+Open [apps]-[Toolbox]-[DVM Withdraw], and click `Connect`
+
+![connect](assets/wiki-tut-dvm-withdraw-01.png)
+
+2. Withdraw
+
+Once connected to MetaMask successfully, fill in the withdrawal address and withdraw amount prepared in advance, click `Withdraw`.
```

Comment:
Once connected to MetaMask, fill in the withdrawal address and amount prepared before